Clinical-Genomics/scout
# Export hpo

fix #1528

**How to test**:
- Go to phenotypes page and make sure that the list can be copied and exported to Excel, same style as causatives

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
